estimation.py

```diff
@@ -16,10 +16,7 @@
 
 from torchtitan.config_manager import JobConfig
 from torchtitan.datasets import build_tokenizer
-from torchtitan.float8_linear import (
```

Comment:
not directly related to this PR: is there a way to share code between `estimation.py` and `train.py`? it's quite painful to manually keep them in sync

Reply:
I agree. People are aware of this -- some context here: https://github.com/pytorch/torchtitan/pull/425#issuecomment-2189788501

Another step I'm going to do is to have a separate training script for PP, which could make it worse.